src/GrumPHP/Collection/FilesCollection.php

```diff
@@ -73,6 +73,22 @@ public function path($pattern)
     }
 
     /**
+     * Filter by paths
+     *
+     * $collection->paths(array('/^spec\/','/^src\/'))
+     *
+     * @param array $patterns
+     *
+     * @return FilesCollection
+     */
+    public function paths(array $patterns)
```

Comment:
Indeed, the configurable options spec from the task should also contain the new option. The last commit is good.
I was talking about the GrumPHP/Collection/FilesCollectionSpec that should contain a new test for the new paths method. Can you add this one?